generators/generator-base.js

```diff
@@ -2447,6 +2447,15 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
         return name + this.dtoSuffix;
     }
 
+    /**
+     * Allows to know if the given field needs an additional content type field to save it's content type.
+     * @param {object} field - the entity field
+     * @returns {boolean} true if it needs an additional field.
+     */
+    isFieldWithContentType(field) {
+        return (field.fieldType === 'byte[]' || field.fieldType === 'ByteBuffer') && field.fieldTypeBlobContent !== 'text';
+    }
+
```

Comment:
@qlereboursBS if we are using methods this should be the place for this method.
But I think we shouldn't add a method for this.
Put `fieldWithContentType` field after: https://github.com/jhipster/generator-jhipster/blob/3ded888de0ba40be807144620e078424e9b06db0/utils/field.js#L160

And use directly.

Reply:
I totally agree, this is the best solution, thanks
I just pushed the fix for this one